src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,195 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.WrongDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private final StorageDao storageDao = new StorageDaoImpl();
+
+    @Test
+    void register_nullPassword_notOk() {
+        User user = new User(123456L, "First", null, 20);
+        assertThrows(WrongDataException.class, () -> registrationService.register(user),
+                "Password cannot be null!");
```

Comment:
In this case you expect to see you CustomException. Therefore, it should be written in the message. You message can be something similar to:
"Expected " + WrongDataException.class.getName() + " to be thrown for the user with null password, but it wasn't"
Please review your other message and correct them. Message must be more informative.